eventsapi/models.py

```diff
@@ -3,6 +3,10 @@
 from uuid import UUID
 
 
+CONTENT_LOADED_V1 = 'content_loaded_v1'
+CONTENT_LOAD_FAILED_V1 = 'content_load_failed_v1'
```

Comment:
@mjoss17 - Ah, I realized there is a simple way to avoid referencing these strings in the first place which will solve the problem. We can discuss / walk through those changes when we meet.